lib/pubid/iso/typed_stage.rb

```diff
@@ -0,0 +1,204 @@
+module Pubid::Iso
+  class TypedStage
+    attr_accessor :type, :stage, :typed_stage
+
+    TYPED_STAGES = {
+      dtr: {
```

Comment:
Later on let’s split this information into individual identifier classes. We have information splattered everywhere about different document types.

Reply:
I think we should really incorporate the Excel table somehow, in each identifier class.